index.bs

```diff
@@ -318,6 +284,62 @@ with the website process. The [=/file system path=] might contain components
 which are not known to the website unless the [=/file system locator=] is later
 [=file system locator/resolved=] relative to a parent [=directory locator=].
 
+<div algorithm>
+<dfn abstract-op id=entry-query-access lt=QueryFileSystemAccess>QueryFileSystemAccess(|locator|, |mode|)</dfn>
```

Comment:
I don't have much experience with abstract algorithms, so please let me know whether it makes sense to use in this case

Reply:
I think `abstract-op` is primarily for JavaScript and some legacy cases. For new cases we can just define an algorithm as we already do elsewhere.

> To query file system access, given ...